json_parsers/json_parser4.py.py

```diff
@@ -0,0 +1,34 @@
+
+import json
+
+    
+class Json:
+
+    def json_convert(self,path):
+        f=open(input_path.strip("\u202a"),'r')
+        data=f.read()
+        y=json.loads(data)
+        f.close()
+        print(y)
+        print('#'*50)
+        z=y["markers"]
+        z[0]["location"]=z[0].pop("position")
+        return y
+    
+    def To_json(self,y):
```

Comment:
1. Usually the function names should start with lower case letters
2. The function should have return statement with proper exceptional handling if require